assets/stylesheets/comments.scss

```diff
@@ -47,6 +47,10 @@
         }
     }
 
+    .openqa-bugref:last-child {
```

Comment:
After discussing we found out that this would only mark one bugref out of potentially multiple within a single comment but we do carry-over of "comments including bugref(s)"

Reply:
I also added a unit test for this now. Since we've been unsure what the exact behavior is more than once.

Reply:
Turns out my fixes for the markdown version of the same code got lost. That's why the tests wouldn't pass. So I thought maybe we can prevent this and merged both code paths.